"Failed to retrieve Windows Defender status." at startup of Eclipse

The Windows Defender status check assumes the result strings are case insensitive.

On line 319, eclipse checks for "Passive mode"
https://github.com/eclipse-platform/eclipse.platform.ui/blame/03b03d8feefa6270e575dd61eb5c25c839c36eec/bundles/org.eclipse.ui.workbench/Eclipse%20UI/org/eclipse/ui/internal/WindowsDefenderConfigurator.java#L319

The error I get on my windows 11 workstation gives us "Passive Mode"
```
org.eclipse.core.runtime.CoreException: Failed to retrieve Windows Defender status.
	at org.eclipse.ui.internal.WindowsDefenderConfigurator.isWindowsDefenderActive(WindowsDefenderConfigurator.java:324)
	at org.eclipse.ui.internal.WindowsDefenderConfigurator.runExclusionCheck(WindowsDefenderConfigurator.java:154)
	at org.eclipse.ui.internal.WindowsDefenderConfigurator.lambda$0(WindowsDefenderConfigurator.java:101)
	at org.eclipse.core.runtime.jobs.Job$2.run(Job.java:187)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.io.IOException: Process terminated with unexpected result:
Passive Mode
	at org.eclipse.ui.internal.WindowsDefenderConfigurator.isWindowsDefenderActive(WindowsDefenderConfigurator.java:321)
	... 4 more
```

--- a/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/WindowsDefenderConfigurator.java
+++ b/bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/WindowsDefenderConfigurator.java
@@ -23,6 +23,7 @@ import java.nio.charset.StandardCharsets;
 import java.nio.file.Path;
 import java.util.Base64;
 import java.util.List;
+import java.util.Locale;
 import java.util.Optional;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.ExecutorService;
@@ -313,11 +314,11 @@ public class WindowsDefenderConfigurator implements EventHandler {
 		try {
 			List<String> lines = runProcess(command, monitor);
 			String onlyLine = lines.size() == 1 ? lines.get(0) : ""; //$NON-NLS-1$
-			return switch (onlyLine) {
+			return switch (onlyLine.toLowerCase(Locale.ENGLISH)) {
 			// Known values as listed in
 			// https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/microsoft-defender-antivirus-windows#use-powershell-to-check-the-status-of-microsoft-defender-antivirus
-			case "SxS Passive Mode", "Passive mode" -> false; //$NON-NLS-1$ //$NON-NLS-2$
-			case "Normal", "EDR Block Mode" -> true; //$NON-NLS-1$//$NON-NLS-2$
+			case "sxs passive mode", "passive mode" -> false; //$NON-NLS-1$ //$NON-NLS-2$
+			case "normal", "edr block mode" -> true; //$NON-NLS-1$//$NON-NLS-2$
 			default -> throw new IOException("Process terminated with unexpected result:\n" + String.join("\n", lines)); //$NON-NLS-1$//$NON-NLS-2$
 			};
 		} catch (IOException e) {

